app/code/Transoft/Blog/Ui/Component/Listing/Column/ContactsActions.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Ui\Component\Listing\Column;
+
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Listing\Columns\Column;
+
+/**
+ * Class ContactsActions
+ *
+ * UI component contacts actions class
+ */
+class ContactsActions extends Column
+{
+    /**
+     * @var UrlInterface
+     */
+    protected $urlBuilder;
+
+    /**
+     * ContactsActions constructor.
+     *
+     * @param ContextInterface $context
+     * @param UiComponentFactory $uiComponentFactory
+     * @param UrlInterface $urlBuilder
+     * @param array $components
+     * @param array $data
+     */
+    public function __construct(
+        ContextInterface $context,
+        UiComponentFactory $uiComponentFactory,
+        UrlInterface $urlBuilder,
+        array $components = [],
+        array $data = []
+    ) {
+        $this->urlBuilder = $urlBuilder;
```

Comment:
Remove. Use \Magento\Framework\View\Element\UiComponent\ContextInterface::getUrl

Reply:
\Magento\Framework\View\Element\UiComponent\ContextInterface method 'getUrl' returns url including 
parameters(which are not expected here), i couldn't find a way to get only base url like 'http:/mysite.local' using this method. 
Urlinterface is now used in 'app/code/Transoft/Blog/ViewModel/Bloglist.php'. Is there a difference between using
this interface within ContactsActions.php and Bloglist.php?

Reply:
fixed